Portfolio2.css

```diff
@@ -0,0 +1,222 @@
+* {
+	margin:0;
+	padding: 0;
+	
+}
+
+body {
+	background-image: url("https://wallpaperaccess.com/full/1128684.jpg");
+	background-size: cover;
+	
+	
+ }
+
+header {
+	display: inline-flex; 
+	position: fixed; 
+	margin-left: 1200px;
+	position:fixed;
+	margin-top:-180px;
+}
+
+.menue ul li {
+	display: inline-flex;
+	list-style: none;
+	cursor:pointer;
+	padding:25px;
+	
+}
+
+.logo img {
+	width: 90px;
+	margin-top: -7px;
+	margin-right: 48px;
+	
+}
+
+.active {
+	color: grey
+}
+.activ {
+	color: grey;
+}
+
+.acti {
+	color: grey;
+}
+
+#headertwo {
+        background-color: none;
+    	width: 100%;
+    	text-align: right;
+	font-size: 1cm;
+	margin-top: 200px;
+	font-family: garamond;
+  	color: #7c7b85;
+	padding: 14px 16px;
+  	text-decoration: none;
+	margin-left: 100px ;
+  	text-align: left;
+  	font-weight: bold;
+  	color:#6e7785;
+	
+	  
+}
+
+.
+
+}
+
+#header h1 {
+	margin-left:50px;
+	
+}
+
+
+
+.sub-title {
+	width: 100%;
+	float: left;
+	
+	
+	}
+
+p {
+	width: 900px;
+	font-family: "Times New Roman";
+	font-size:20px;
+	line-height:25px;
+	color: white;
+	margin-right: 70px;
+}
+
+
+	
+
+.column {
+	overflow: auto;
+}
+
+.threecolumn {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	margin-bottom: 100px;
+	margin-left: 100px;
+	
+}
+
+.fourcolumns {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-rows: 1fr 1fr;
+	margin-bottom: 100px;
+	margin-left: 100px;
```

Comment:
Same comment as before, remove the margin left and do the same thing you did to center your banners

Reply:
This still seems to be an issue, there is still the margin-left: 100px to position it rather than the margin: 0 auto; width: 90% method you've used elsewhere.